sympy/core/expr.py

```diff
@@ -1956,6 +1956,11 @@ def extract_multiplicatively(self, c):
             x = self.extract_multiplicatively(a)
             if x is not None:
                 return x.extract_multiplicatively(b)
+
+        from sympy.core.function import _coeff_isneg as f
```

Comment:
yeah..i tried keeping the import statement at the beginning of the file but it introduced a cyclic dependency error.
have a look here
![bug](https://cloud.githubusercontent.com/assets/17480353/23784554/c2e2c622-0588-11e7-8da4-4d5d0dde0387.png)
To avoid this , i kept the ```import``` within this function.